diff -ruN /home/creichen/src/eclipse/plugins/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/DebugUIPlugin.java ui/org/eclipse/debug/internal/ui/DebugUIPlugin.java
--- /home/creichen/src/eclipse/plugins/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/DebugUIPlugin.java	2007-09-21 10:58:30.000000000 -0600
+++ ui/org/eclipse/debug/internal/ui/DebugUIPlugin.java	2008-10-24 01:05:17.000000000 -0600
@@ -17,7 +17,7 @@
 import java.lang.reflect.InvocationTargetException;
 import java.net.URL;
 import java.util.HashSet;
-import java.util.Iterator;
+//import java.util.Iterator;
 import java.util.Set;
 
 import javax.xml.parsers.DocumentBuilder;
@@ -30,13 +30,13 @@
 import javax.xml.transform.dom.DOMSource;
 import javax.xml.transform.stream.StreamResult;
 
-import org.eclipse.core.resources.ISaveContext;
+//import org.eclipse.core.resources.ISaveContext;
 import org.eclipse.core.resources.ISaveParticipant;
 import org.eclipse.core.resources.IncrementalProjectBuilder;
 import org.eclipse.core.resources.ResourcesPlugin;
 import org.eclipse.core.runtime.CoreException;
 import org.eclipse.core.runtime.FileLocator;
-import org.eclipse.core.runtime.IAdapterManager;
+//import org.eclipse.core.runtime.IAdapterManager;
 import org.eclipse.core.runtime.IConfigurationElement;
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
@@ -50,15 +50,16 @@
 import org.eclipse.core.runtime.jobs.Job;
 import org.eclipse.debug.core.DebugException;
 import org.eclipse.debug.core.DebugPlugin;
-import org.eclipse.debug.core.IExpressionManager;
+//import org.eclipse.debug.core.IExpressionManager;
 import org.eclipse.debug.core.ILaunch;
 import org.eclipse.debug.core.ILaunchConfiguration;
-import org.eclipse.debug.core.ILaunchConfigurationType;
+//import org.eclipse.debug.core.ILaunchConfigurationType;
 import org.eclipse.debug.core.ILaunchConfigurationWorkingCopy;
 import org.eclipse.debug.core.ILaunchListener;
-import org.eclipse.debug.core.ILaunchManager;
+//import org.eclipse.debug.core.ILaunchManager;
 import org.eclipse.debug.core.IStatusHandler;
 import org.eclipse.debug.core.Launch;
+/*
 import org.eclipse.debug.core.model.IDebugElement;
 import org.eclipse.debug.core.model.IDebugTarget;
 import org.eclipse.debug.core.model.IExpression;
@@ -69,8 +70,9 @@
 import org.eclipse.debug.core.model.IStackFrame;
 import org.eclipse.debug.core.model.IThread;
 import org.eclipse.debug.core.model.IVariable;
+*/
 import org.eclipse.debug.internal.ui.contextlaunching.LaunchingResourceManager;
-import org.eclipse.debug.internal.ui.contexts.SuspendTriggerAdapterFactory;
+//import org.eclipse.debug.internal.ui.contexts.SuspendTriggerAdapterFactory;
 import org.eclipse.debug.internal.ui.launchConfigurations.ClosedProjectFilter;
 import org.eclipse.debug.internal.ui.launchConfigurations.DeletedProjectFilter;
 import org.eclipse.debug.internal.ui.launchConfigurations.LaunchConfigurationEditDialog;
@@ -81,13 +83,13 @@
 import org.eclipse.debug.internal.ui.launchConfigurations.PerspectiveManager;
 import org.eclipse.debug.internal.ui.sourcelookup.SourceLookupFacility;
 import org.eclipse.debug.internal.ui.sourcelookup.SourceLookupManager;
-import org.eclipse.debug.internal.ui.stringsubstitution.SelectedResourceManager;
+//import org.eclipse.debug.internal.ui.stringsubstitution.SelectedResourceManager;
 import org.eclipse.debug.internal.ui.views.breakpoints.BreakpointOrganizerManager;
-import org.eclipse.debug.internal.ui.views.breakpoints.OtherBreakpointCategory;
+//import org.eclipse.debug.internal.ui.views.breakpoints.OtherBreakpointCategory;
 import org.eclipse.debug.internal.ui.views.console.ProcessConsoleManager;
-import org.eclipse.debug.internal.ui.views.launch.DebugElementAdapterFactory;
+//import org.eclipse.debug.internal.ui.views.launch.DebugElementAdapterFactory;
 import org.eclipse.debug.internal.ui.views.launch.DebugElementHelper;
-import org.eclipse.debug.internal.ui.views.memory.renderings.MemorySegment;
+//import org.eclipse.debug.internal.ui.views.memory.renderings.MemorySegment;
 import org.eclipse.debug.ui.DebugUITools;
 import org.eclipse.debug.ui.IDebugModelPresentation;
 import org.eclipse.debug.ui.IDebugUIConstants;
@@ -467,6 +469,7 @@
 	 */
 	public void start(BundleContext context) throws Exception {
 		super.start(context);
+		/*
 		ResourcesPlugin.getWorkspace().addSaveParticipant(this,
 				new ISaveParticipant() {
 					public void saving(ISaveContext saveContext) throws CoreException {
@@ -542,7 +545,7 @@
 		fServiceTracker = new ServiceTracker(context, PackageAdmin.class.getName(), null);
 		fServiceTracker.open();
 		fPackageAdminService = (PackageAdmin) fServiceTracker.getService();
-		
+		*/
 		getLaunchConfigurationManager().startup();
 	}
 
diff -ruN /home/creichen/src/eclipse/plugins/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/sourcelookup/Prompter.java ui/org/eclipse/debug/internal/ui/sourcelookup/Prompter.java
--- /home/creichen/src/eclipse/plugins/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/sourcelookup/Prompter.java	2007-09-21 11:02:28.000000000 -0600
+++ ui/org/eclipse/debug/internal/ui/sourcelookup/Prompter.java	2008-10-24 01:11:38.000000000 -0600
@@ -40,6 +40,7 @@
 	 * @see org.eclipse.debug.core.IStatusHandler#handleStatus(org.eclipse.core.runtime.IStatus, java.lang.Object)
 	 */
 	public Object handleStatus(final IStatus status, final Object source) throws CoreException {
+		if (true) return new Boolean(true);
 		DebugPlugin dp = DebugPlugin.getDefault();
 		// on shutdown the debug plug-in can be null
 		if (dp == null) {
